problems/arraysWithLoops.js

```diff
@@ -25,39 +38,71 @@ function shoutWhileLoop() {}
  * @returns {Array} - ["I!", "am!", "a!", "happy!", "array!"]
  */
 
-function shoutForOfLoop() {}
+const shoutForOfLoop = (arr) => {
+  let newArray = []
+  for(let element of arr){
+    newArray.push(element + "!");
+  }
+return newArray;
+}
+  
 
 /**
  * Takes in an array and returns the sum of all values
  * @param {number[]} nums
  * @returns {number} sum
  */
 
-function sumArray() {}
+const sumArray = (nums) => {
+  let sum = 0; 
+        for(let i = 0; i < nums.length; i++ ) {
+            sum += nums[i];
+        }    
+    return sum;
+}
 
 /**
  * Takes in an array and returns an array of all the odd valued elements
  * @param {number[]} nums
  * @returns {array} odds
  */
 
-function oddArray() {}
+const oddArray = (nums) => {
+  let odds = []
+  for (let i = 0; i < nums.length; i++) {
+    if (nums[i] % 2 === 0) continue;
```

Comment:
if you change to === 1 you can do the logic below and avoid the continue all together.